src/jit/codegenarm64.cpp

```diff
@@ -5723,17 +5750,163 @@ void CodeGen::genLeaInstruction(GenTreeAddrMode *lea)
     genProduceReg(lea);
 }
 
+/*****************************************************************************
+ *  The condition to use for (the jmp/set for) the given type of compare operation are
+ *  returned in 'jmpKind' array.  
+ *  If only one branch is necessary the value of jmpKind[1] will be EJ_NONE
+ *  On Arm64 both branches will always branch to the true label
+ */
```

Comment:
This was copied from CodegenXarch and then edited.
I will change the both of these to use a new  standard function header.